nupic/engine/__init__.py

```diff
@@ -29,6 +29,22 @@
 
 basicTypes = ['Byte', 'Int16', 'UInt16', 'Int32', 'UInt32', 'Int64', 'UInt64', 'Real32', 'Real64', 'Handle']
 
+pyRegions = {"nupic.regions.AnomalyRegion": "AnomalyRegion",
+           "nupic.regions.CLAClassifierRegion": "CLAClassifierRegion",
```

Comment:
and this doesn't really need to be a dictionary since you just want to iterate over the elements. You could make it a tuple of tuples which would be immutable:

```
pyRegions = (
    ("nupic.regions.AnomalyRegion", "AnomalyRegion"),
    ...
)
```

and later:

```
for module, className in pyRegions:
```